examples/templates/slack-template.twig

```diff
@@ -8,11 +8,14 @@
 {% endif %}
 
 {{ salutation }}, please have a look
+
 {% for ticket in tickets %}
     {% set status = ticket.status %}
     {% set daysDiff = status.changeDate.diff(now).days %}
     {% set url = "https://" ~ companyName ~ ".atlassian.net/browse/" ~ ticket.key %}
-    {% set dayWord = (daysDiff > 1) ? 'days' : 'day' %}> [<{{ url }}|{{ ticket->key() }}>] {{ ticket->title() }}
```

Comment:
@JesusValera Just as a note for any future twig usage:  
Instead of `ticket->key()` you should write `ticket.key` 🎋 💯